sw/device/tests/sram_ctrl_execution_test_ret.c

```diff
@@ -93,7 +93,7 @@ typedef void (*sram_ctrl_instruction_fault_function_t)(void);
  * https://github.com/riscv-non-isa/riscv-asm-manual/blob/master/riscv-asm.md
  */
 static void sram_execution_test(void) {
-  memset((void *)kRetRamStartAddr, 0, SRAM_CTRL_DATA_NUM_BYTES);
+  memset((void *)kRetRamStartAddr, 0, SRAM_CTRL_TESTUTILS_DATA_NUM_BYTES);
```

Comment:
You never need `void*` casts in C.

Reply:
(`kRetRamStartAddr` is a `uint32_t` so there does need to be a cast to a pointer somewhere)